lib/api/constants.js

```diff
@@ -0,0 +1,5 @@
+const constants = {
+    locationStatusCollection: '__locationStatusStore',
```

Comment:
does not seem related to "api", i would expect to find this closer to database : is there some "central" place where we already define the name of the metastore or other system collections?

Reply:
We don't have anything like that actually, the `metastore` collection name for example is hardcoded into the mongo client in arsenal